### DISCUSSION TITLE
[Question] How to increase the maximum usage of GPU in the training(currently only 30% occupied and can not increase)

### DISCUSSION THREAD

**Discussion by calmdw** [isaac-sim/IsaacLab#1139] [Q&A]
### Question
Hi, I am trying to train the object reach task with customized robot. I open 4096 environments but the rendering is slow and the usage of GPU only 30% and can not increase anymore even opening more environments.

**Comment by glvov-bdai** [comment]
Hi, you can see an example of how to maximize the number of parallel environments here: https://isaac-sim.github.io/IsaacLab/source/how-to/estimate_how_many_cameras_can_run.html

For a large number of environments, it is recommended to use headless mode. You can spot check trained checkpoints through the play functionality with a lower number of environments and the rendering will be faster.

If these ideas don't work for you, please provide more detailed information about how to recreate your scenario as a standalone file, and also your system information.
